arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/EditFeaturesWithFeatureLinkedAnnotationViewController.swift

```diff
@@ -0,0 +1,253 @@
+// Copyright © 2020 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class EditFeaturesWithFeatureLinkedAnnotationViewController: UIViewController {
+    @IBOutlet weak var mapView: AGSMapView! {
+        didSet {
+            // Create the map with a light gray canvas basemap centered on Loudoun, Virginia.
+            mapView.map = AGSMap(basemapType: .lightGrayCanvasVector, latitude: 39.0204, longitude: -77.4159, levelOfDetail: 18)
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+            loadGeodatabase()
+        }
+    }
+    // The feature that has been selected.
+    var selectedFeature: AGSFeature?
+    // A geodatabase to be loaded.
+    var geodatabase: AGSGeodatabase?
+    // The returned cancelable after identifying the layers.
+    var identifyLayers: AGSCancelable?
+    
+    func loadGeodatabase() {
+        // Obtain the geodatabase URL from portal data.
+        let geodatabaseURL = Bundle.main.url(forResource: "loudoun_anno", withExtension: "geodatabase")!
+        geodatabase = AGSGeodatabase(fileURL: geodatabaseURL)
+        guard let geodatabase = geodatabase else { return }
```

Comment:
We know that the geodatabase isn't `nil` since it was just created, so it would make more sense to force unwrap than to guard here. But why is the method even dealing with an optional? Instead of assigning the geodatabase to the property and then accessing the property, it would be better to assign the geodatabase to a local, non-optional variable and then use that variable within the method, including to set the property's value. Or perhaps even better, just make the geodatabase property non-optional and immutable by assigning it a value where it is declared.